First screen

amaurycb opened the pull request:
navigation for the second fragment

amaurycb committed:
Create First Screen

amaurycb committed:
Create First Screen

amaurycb committed:
Create First Screen

williamgdev committed:
Fixed Adapter

amaurycb committed:
.

amaurycb committed:
FirstScreen

williamgdev commented on app/src/main/java/com/example/appempty/SecondFragment.kt:
```diff
@@ -27,16 +27,17 @@ class SecondFragment : Fragment() {
-           username.text = result.login?.username ?: "No User Name"
-           state.text = result.location?.state ?: "No State"
-           email.text = result.email
+          fullname.text = result[0].name?.first ?: "No First Name"
```
create a val for result[0], you should not be making this is a bad practice, if someone changes to 1 you will need a magnifier to find out where is the wrong index. instead of result[0] you can use result.first()

amaurycb replied:
Done

williamgdev commented on app/src/main/java/com/example/appempty/ViewModel/UserListViewModel.kt:
```diff
@@ -0,0 +1,41 @@
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+class UserListViewModel {
```
This is not been used in the app, why we need this file for?

williamgdev commented on app/src/main/java/com/example/appempty/ViewModel/UserViewModel.kt:
```diff
@@ -38,7 +38,7 @@ class UserViewModel : ViewModel() {
 
                 val result: UserProfile = response.body()?.results?.firstOrNull() ?: UserProfile()
-                usuario.value = result
+                usuario.value = MutableList(100) { result }
```
This should happen in the UserList, that is why you used result[0] and did not use the listaViewModel, because you are using only one ViewModel for both fragments, both logics should be on a separate ViewModels and attached to both Fragments.

williamgdev reviewed (changes requested):
I ran the app and did not see the RecyclerView as a Grid like the design attached to the ticket.
<img src="https://trello-attachments.s3.amazonaws.com/5f272753c802463214554451/720x540/c9467a1183ce269165baee6cd9375042/image.png" height ="700"/>

amaurycb committed:
Updata 1

amaurycb committed:
create navigation tablet

amaurycb committed:
create navigation tablet

amaurycb committed:
update UserListViewModel.kt and ListaFragment.kt

amaurycb committed:
Line 26 genera  error  con count = 10  y entra en  onFailure
Line 38 tambien con problemas

En ListaFragments pendiente ajustar line 39 para seleccionar que layout usar pero ni idea

amaurycb committed:
En UserViewModel Line 38 tambien tiene su problema

En ListaFragments pendiente ajustar line 38 para navegar al 2do fragment solo si esta en portrain. En caso contrario no responder al click
